plugins/jobs/src/js/data/__tests__/JobModel-test.js

```diff
@@ -246,7 +246,7 @@ describe("JobModel Resolver", () => {
           marbles(m => {
             m.bind();
 
-            const result$ = oneJobsResponse(m, null, "STATUS", "ASC").map(
+            const result$ = oneJobsResponse(m, undefined, "STATUS", "ASC").map(
```

Comment:
I am just curious on why would we prefer undefined here, instead of null.

Reply:
I explained my point of view here: https://github.com/dcos/dcos-ui/pull/3030#discussion_r198786706
`undefined` = what the word says, "nothing" or "not passed" (as a parameter)
`null` = not _yet_ set, but will be set soon and is not supposed to return to value `null` after that.

in addition to that, dealing with `undefined` is way easier when it comes to defaults, because  `null` will not cause the default to be used (see changes in `JobConnection` TypeResolver)